src/legacy/core_plugins/region_map/public/components/region_map_options.tsx

```diff
@@ -0,0 +1,253 @@
+              defaultMessage="Layer settings"
+            />
+          </h2>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+
+        <SelectOption
+          id="regionMapOptionsSelectLayer"
+          label={i18n.translate('regionMap.visParams.vectorMapLabel', {
+            defaultMessage: 'Vector map',
+          })}
+          labelAppend={
+            stateParams.emsHotLink && (
+              <EuiText size="xs">
+                <EuiLink
+                  href={stateParams.emsHotLink}
+                  target="_blank"
+                  rel="noopener"
+                  title={i18n.translate('regionMap.visParams.previewOnEMSLinkTitle', {
+                    defaultMessage: 'Preview {selectedLayerName} on the Elastic Maps Service',
+                    values: {
+                      selectedLayerName:
+                        stateParams.selectedLayer && stateParams.selectedLayer.name,
+                    },
+                  })}
+                >
+                  <FormattedMessage
+                    id="regionMap.visParams.previewOnEMSLinkText"
+                    defaultMessage="Preview on EMS"
+                  />{' '}
+                  <EuiIcon type="popout" size="s" />
+                </EuiLink>
+              </EuiText>
+            )
+          }
+          options={vectorLayerOptions}
+          paramName="selectedLayer"
+          value={stateParams.selectedLayer && stateParams.selectedLayer.layerId}
+          setValue={setLayer}
+        />
+
+        <SelectOption
+          id="regionMapOptionsSelectJoinField"
+          label={i18n.translate('regionMap.visParams.joinFieldLabel', {
+            defaultMessage: 'Join field',
+          })}
+          options={fieldOptions}
+          paramName="selectedJoinField"
+          value={stateParams.selectedJoinField && stateParams.selectedJoinField.name}
+          setValue={setField}
+        />
+
+        <SwitchOption
+          label={i18n.translate('regionMap.visParams.displayWarningsLabel', {
+            defaultMessage: 'Display warnings',
+          })}
+          tooltip={i18n.translate('regionMap.visParams.switchWarningsTipText', {
+            defaultMessage:
+              'Turns on/off warnings. When turned on, warning will be shown for each term that cannot be matched to a shape in the vector layer based on the join field. When turned off, these warnings will be turned off.',
+          })}
+          paramName="isDisplayWarning"
+          value={stateParams.isDisplayWarning}
+          setValue={setValue}
+        />
+
+        <SwitchOption
+          label={i18n.translate('regionMap.visParams.showAllShapesLabel', {
+            defaultMessage: 'Show all shapes',
+          })}
+          tooltip={i18n.translate('regionMap.visParams.turnOffShowingAllShapesTipText', {
+            defaultMessage:
+              'Turning this off only shows the shapes that were matched with a corresponding term',
```

Comment:
Let's add a dot at the end of tooltips, just to be consistent:
```suggestion
              'Turning this off only shows the shapes that were matched with a corresponding term.',
```